### ISSUE TITLE
How to mask the LocalHost URL to a custom url

### ISSUE THREAD

**Issue by soforeal** [Telerik-Verified-Plugins/WKWebView#204]
I am currently using WKWebView and a gelocation function in my cordova app using the HTML5 geolocation (which refreshes faster in my app than the cordova geolocation plugin). 

When the app asks the user for permission to view their location, the string reads "http://localhost:12344 would like to use your current location". 

Is there any way to mask that URL so it reads "appName would like to use your current position"? I'd even settle for "http://localhost/appName"!

Thanks for such an awesome plugin!

**Comment by EddyVerbruggen** [comment]
I don't think you can if you use HTML5 geolocation, but perhaps others have a solution?
